swim/join_sender.go

```diff
@@ -112,6 +130,9 @@ func newJoinSender(node *Node, opts *joinOpts) (*joinSender, error) {
 		node: node,
 	}
 
+	// Parse bootstrap hosts into a map
```

Comment:
Agreed, I think that's a good pattern. I'm gonna note that down as a TODO item for Refactor Week™.